sdk/attestation/azure-security-attestation/src/attestation_client.cpp

```diff
@@ -212,7 +213,7 @@ Azure::Response<TpmAttestationResult> AttestationClient::AttestTpm(
     // Send the request to the service.
     auto response
         = AttestationCommonRequest::SendRequest(*m_pipeline, request, tracingContext.Context);
-    std::string returnedBody(TpmDataSerializer::Deserialize(response));
+    auto returnedBody = TpmDataSerializer::Deserialize(response);
```

Comment:
C++ guidelines discourage the use of assignment initialization in favor of either ctor syntax (`returnedBody(TpmDataSerializer::Deserialize(response))`) or initializer syntax (`returnedBody{TpmDataSerializer::Deserialize(response)}`).